target_nav_node:
  ros__parameters:
    front_angle_center_deg: 180.0
